.github/workflows/sdk-release.yaml

```diff
@@ -141,3 +141,28 @@ jobs:
         if: ${{ failure() }}
         working-directory: docker/compose/validator-testnet
         run: docker logs local-testnet
+
+  # Run the TS SDK indexer tests. Note: indexer service can be flaky and we
+  # dont want those tests to be land blocking for any PR on the aptos repo.
+  # This is why we run those tests separate from
+  # test-sdk-confirm-client-generated-publish.
+  run-indexer-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
can you pin the version of these third party action deps like in the other job? e.g.
```
actions/setup-node@969bd2663942d722d85b6a8626225850c2f7be4b # pin@v3
```

Reply:
should I do the same for checkout?
```
actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3
```

Reply:
yep, and `pnpm/action-setup@v2` as well

Reply:
```
pnpm/action-setup@537643d491d20c2712d11533497cb47b2d0eb9d5 # pin https://github.com/pnpm/action-setup/releases/tag/v2.2.3
```

Reply:
we use it like this in the other job thought https://github.com/aptos-labs/aptos-core/pull/8454/files#diff-f9bdc9424e6b94fc4cdf8b6974e1c90d62ffc04c88d91c80c721022b183df298R62-R64

Reply:
ah that should best be changed as well. pinning the third party dep by SHA is a best practice